di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -107,52 +109,67 @@ public class IpvHandler {
 
     private String getAuthorizeRedirect(QueryParamsMap queryMap, String errorType)
             throws Exception {
+
         var environment = queryMap.get(ENVIRONMENT_PARAM).value();
         var userIdTextValue = queryMap.get(USER_ID_PARAM).value();
         var signInJourneyIdText = queryMap.get(JOURNEY_ID_PARAM).value();
-        var vtr =
-                Arrays.stream(queryMap.get(VTR_PARAM).value().split(","))
-                        .map(String::trim)
-                        .filter(value -> !value.isEmpty())
-                        .toList();
         var userEmailAddress = queryMap.get(EMAIL_ADDRESS_PARAM).value();
-        var reproveIdentityString = queryMap.get(REPROVE_IDENTITY_PARAM).value();
-        var reproveIdentityClaimValue =
-                StringUtils.isNotBlank(reproveIdentityString)
-                        ? JwtBuilder.ReproveIdentityClaimValue.valueOf(reproveIdentityString)
-                        : JwtBuilder.ReproveIdentityClaimValue.NOT_PRESENT;
-
-        var includeInheritedId =
-                Objects.equals(queryMap.value(INHERITED_ID_INCLUDED_PARAM), "checked");
-        var inheritedIdVot = queryMap.get(INHERITED_ID_VOT_PARAM).value();
-        var inheritedIdSubject = queryMap.value(INHERITED_ID_SUBJECT_PARAM);
-        var inheritedIdEvidence = queryMap.value(INHERITED_ID_EVIDENCE_PARAM);
-
         var userId = getUserIdValue(userIdTextValue);
-
-        JWTClaimsSet claims =
-                JwtBuilder.buildAuthorizationRequestClaims(
-                        userId,
-                        signInJourneyIdText,
-                        ORCHESTRATOR_STUB_STATE.getValue(),
-                        vtr,
-                        errorType,
-                        userEmailAddress,
-                        reproveIdentityClaimValue,
-                        environment,
-                        includeInheritedId,
-                        inheritedIdSubject,
-                        inheritedIdEvidence,
-                        inheritedIdVot);
+        var isMfaReset = Objects.equals(queryMap.value(MFA_RESET_PARAM), "checked");
```

Comment:
Might be worth making "checked" static final. Looks like it's used in a couple of places.